src/components/tab-panel/PermissionsTabPanel.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <q-tab-panel
+    name="permissions"
+    data-cy="permissions_tab_panel"
+  >
+    <h6
+      class="q-ma-none q-mb-sm"
+      data-cy="permissions_title"
+    >
+      {{ $t('PermissionsTabPanel.text.title', { name: entity.name }) }}
+    </h6>
+    <q-banner
+      v-if="isSuperAdmin"
+      dense
+      class="bg-warning text-white text-weight-bold q-mb-md"
+      :data-cy="`tab_${type}_warning`"
+    >
+      <template #avatar>
+        <q-icon
+          :name="$t('PermissionsTabPanel.icon.warning')"
+        />
+      </template>
+      {{ warningText }}
+    </q-banner>
+    <q-btn
+      v-if="showAttachDetachButton && !isSuperAdmin"
+      outline
+      no-caps
+      color="primary"
+      class="bg-white q-mb-md"
+      data-cy="button_attach_permission"
+      :label="$t('PermissionsTabPanel.text.attach')"
+      :icon="$t('PermissionsTabPanel.icon.attach')"
+      @click="openAttachDialog"
+    />
+    <permissions-table
+      v-model:filter-entity="entityName"
+      v-model:filter-action="actionName"
+      v-model:filter-library-id="libraryId"
+      v-model:current-page="currentPage"
+      v-model:max-page="maxPage"
+      v-model:elements-per-page="elementsPerPage"
+      v-model:total-elements="totalElements"
+      :permissions="permissions"
+      :show-action="false"
+      :remove-action="false"
+      :detach-action="showAttachDetachButton && !isSuperAdmin"
+      :no-data-label="$t('PermissionsTable.text.noData')"
+      :no-data-icon="$t('PermissionsTable.icon.noData')"
+      @detach="(event) => isSuperAdmin ? false : openDetachDialog(event)"
+      @on-filter="search"
+    />
+  </q-tab-panel>
+</template>
+
+<script setup>
+import {
+  onMounted,
+  onUnmounted,
+  ref,
+  watch,
+} from 'vue';
+import * as PermissionService from 'src/services/PermissionService';
+import ReloadPermissionsEvent from 'src/composables/events/ReloadPermissionsEvent';
+import DialogEvent from 'src/composables/events/DialogEvent';
+import PermissionsTable from 'src/components/tables/PermissionsTable.vue';
+
+const props = defineProps({
+  entity: {
+    type: Object,
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+  },
+  isSuperAdmin: {
+    type: Boolean,
+    default: false,
+  },
+  warningText: {
+    type: String,
+    default: '',
+  },
+  showAttachDetachButton: {
```

Comment:
I think is useless property, because you can calculate it from the `type` like that:
```js
const showAttachDetachButton = computed(() => !isSuperAdmin.value && type === 'role');
```

Reply:
done